lib/teamcity/postgresql/License_System.Threading.Tasks.Extensions.txt

```diff
@@ -0,0 +1,128 @@
+
+MICROSOFT SOFTWARE LICENSE TERMS
+
+
+MICROSOFT .NET LIBRARY 
```

Comment:
We need that dependency for testing Npgsql v3. So I have put it there, but this causes NHibernate to redistribute it and so being subject to comply with that licence.

Maybe we should instead add the NuGet dependency in the test project, it would avoid having to "redistribute" that only for tests.